app/assets/javascripts/quby/answers/validation.js

```diff
@@ -189,6 +208,59 @@
     return !failed;
   };
 
+  function parsePartialDate(inputs) {
+    var values = partialDateValues(inputs)
+    if(!dateValuesValid(values)) {
+      throw "invalidDate";
+    }
+    else {
```

Comment:
Als de if een throw doet hoef je geen else te doen. 
Zonder else maakt het duidelijker dat het een guard is, en niet een valide branch.